examples/custom-grpc-network/README.md

```diff
@@ -0,0 +1,97 @@
+# Custom Network Configuration with gRPC Web Proxies
+
+This guide demonstrates how to configure the Hedera SDK to communicate with a custom network using gRPC web proxies in a React application. The example sets up a transfer transaction, which is triggered when the page is loaded, using a custom network configuration with multiple gRPC web proxies.
+
+## Steps to Configure the Custom Network
+
+### 1. Setup the Operator Account
+
+First, you need to set up your operator account and private key. These credentials are required to sign and authorize the transaction.
+
+```javascript
+const operatorId = AccountId.fromString("0.0.1458");
+const operatorKey = PrivateKey.fromStringECDSA(
+    "3030020100300706052b8104000a042204208c4ffe487636e8102f72502a2ed16cb4f98073582a777fcfa758481485d9f887",
+);
+```
+
+Replace `0.0.1458` and the private key with your own operator account ID and key.
+
+### 2. Create a Custom Network with gRPC Web Proxies
+
+Define a list of gRPC web proxies to communicate with your custom network. Each entry consists of a proxy URL and its associated account ID.
+
+```javascript
+const nodes = {
+    "https://testnet-node02-00-grpc.hedera.com:443": new AccountId(5),
+    "https://testnet-node03-00-grpc.hedera.com:443": new AccountId(6),
+    "https://testnet-node04-00-grpc.hedera.com:443": new AccountId(7),
+};
+```
+
+These proxy URLs correspond to the gRPC web proxies you wish to use for the custom network.
+
+### 3. Setup the Client Using Client.forNetwork()
+
+To configure the client with the custom network, use `Client.forNetwork()` and pass the list of nodes you defined. The SDK automatically detects the environment (whether it's a browser or Node.js environment) and selects the appropriate implementation (NodeClient, WebClient, or NativeClient).
+
+```javascript
+const client = Client.forNetwork(nodes);
+client.setOperator(operatorId, operatorKey);
+```
+
+In the browser environment, this will use gRPC web proxies.
+
+### 4 Configure the Transaction for the Custom Network
+
+Before sending a transaction, you need to configure it with the necessary parameters, such as the sender, receiver, and amount. Here's how to create a transfer transaction using the configured client for the custom network.
+
+```javascript
+const transferTransaction = new TransferTransaction()
+    .addSender(senderAccountId, amountToSend)
+    .addReceiver(receiverAccountId, amountToReceive)
+    .setTransactionMemo("Transfer via custom gRPC web proxy network")
+    .setTransactionFee(transactionFee);
+
+const response = await transferTransaction.execute(client);
+```
+
+### 5. Running the Application
+
+Once you've set up the client and transaction as described above, you can run your React application with `npm start`. The transfer transaction will be executed using the **custom gRPC web proxies** configured in the Client.forNetwork() method.
+
+## Web Proxies related logs:
+
```

Comment:
I am not 100% convinced we should keep this section, but if we decide to keep it, somewhere we have to note that these are example logs and it's not guarantee the person who setups the app will receive the same response. For example his transaction may be executed against another node and not node 7

Reply:
You're right. This explanation was intended for Simi, so I'll remove it. Additionally, when a developer starts the page, they'll see this image.
<img width="822" alt="image" src="https://github.com/user-attachments/assets/b3496365-3e4d-439d-af0b-cc86eacb4f62" />